src/System.Runtime/tests/System/StringTests.netcoreapp.cs

```diff
@@ -283,6 +286,49 @@ public static void Contains_Match_Char()
             }
         }
 
+        [Fact]
+        public static void Contains_ZeroLength_Char()
+        {
+            ReadOnlySpan<char> sp = new ReadOnlySpan<char>(Array.Empty<char>());
```

Comment:
Can we also add string.Contains asserts here, to expand coverage with new test cases for that as well?

Reply:
Yes. The same would apply to other test cases.